cwf/cloud/helm/cwf-orc8r/templates/analytics.service.yaml

```diff
@@ -14,7 +14,7 @@
 {{- include "orc8rlib.service" (list . "analytics.service") -}}
 {{- define "analytics.service" -}}
 metadata:
-  name: {{ .Release.Name }}-analytics
+  name: analytics
```

Comment:
what do you think about prepending a hardcoded orc8r- or magma- prefix to all of these in case of collision? since a lot of our services are just normal words